[1] - Dashboard(feat): Adding Features for Dashboard Page

akash-kumar-dev opened the pull request:
Addresses: https://github.com/a2i-code-For-Govstack/nise3-skill-and-certification/issues/1

Added features:
- Institute Analytics Cards
- quick display Events Section with calender

akash-kumar-dev commented:
- Added EventSeeder in DatabaseSeeder for automatic seeding Events during migrations

akash-kumar-dev commented:
- Added Trainee Analytics cards (total Trainee, Trainee courses pending Requests, Trainee Certificates requests)
- Added User roles

Shelly011s commented:
- Check for the dashboard link that show error 403 and fix that.
- Check for sidebar links that show 403 error and fix that.
- Fix a grey dot next to the text "Dashboard" which routes to dashboard. Also, make dashboard text itself clickable and make it route to dashboard. Hover should highlight the text as a link and cursor should be pointer.
- Institute admin should not be shown total institutes and total courses, branches, etc should be with respect to that institute.
-  Replace 'management' words to 'analytics' on dashboard for headings. Also change `<h2>` to `<h3>` tags and add padding of `p-1`.
-  Show Institute -> Trraining center -> branch -> batch -> trainee analytics on dashboard

akash-kumar-dev commented:
- Rechecked all sidebars menus and dashboard cards related to error 403
- fixed position of grey dot and changed dashboard to clickable and also added hover effect that can highlight the Text as a link.
- removed total institutes, total courses, branches, .. crads from the Institute Dashboard.
- Replaced 'management' words to 'analytics' on dashboard for headings and Also changed  `<h2>` to` <h3>` tags and added padding of `p-1`
- changed the orders of cards as mentioned above

Shelly011s commented on resources/views/layouts/partials/master/header.blade.php:
```diff
@@ -9,11 +9,17 @@
         @if(\App\Helpers\Classes\AuthHelper::checkAuthUser())
             <li class="nav-item">
-                <a class="nav-link" href="{{route('admin.dashboard')}}" style="padding-top: 12px"><i class="fa fa-circle"></i></a>
+                <a class="nav-link" href="{{route('admin.dashboard')}}" style="padding-top: 12px; left: -20px"><i class="fa fa-circle"></i></a>
```
For the grey dot just remove  `<a>` tag or just `<i> `

akash-kumar-dev commented:
- removed grey dot icon
